lib/clib/Corres_UL_C.thy

```diff
@@ -820,6 +820,15 @@ lemma ccorres_call:
   apply simp
   done
 
+text \<open>This rule is intended to be used in the case where the C calls a function and then uses the
```

Comment:
Style for text comments is different, see style guide.
First paragraph great, second paragraph I'm confused by "allows us to establish a
nontrivial relation between the values we set, if we wish" - what is "values we set" here?

Reply:
We get a value and then set the value. It's easiest to see in the Haskell: it's the `val` return value that the `getter` gets, which is then passed to `setter` which takes it as an argument. Same sort of deal, but of course more complicated, in the C